CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+- Action Points are no longer refreshed on defeated opponents
```

Comment:
```suggestion
- Action Points are no longer refreshed on defeated combatants
```

This also affects player characters, so this would be more clear.